packages/client/components/Poll/local/newPoll.ts

```diff
@@ -0,0 +1,101 @@
+import {commitLocalUpdate, RecordSourceProxy} from 'relay-runtime'
+import clientTempId from '../../../utils/relay/clientTempId'
+import Atmosphere from '../../../Atmosphere'
+import getDiscussionThreadConn from '../../../mutations/connections/getDiscussionThreadConn'
+import safePutNodeInConn from '../../../mutations/handlers/safePutNodeInConn'
+
+const LocalPollOptionsNumber = 2
+
+export const isLocalPoll = (threadable: {id: string}) =>
+  threadable.id.startsWith('poll') && threadable.id.endsWith('tmp')
+
+export const createLocalPoll = (
```

Comment:
In my mind tasks and polls are quite similar so it seems odd to me why polls are created locally and only shared when final, but tasks are created immediately. Maybe you can explain why?